ios/Classes/BetterPlayerPlugin.m

```diff
@@ -85,6 +86,36 @@ - (void)onPlayerSetup:(BetterPlayer*)player
     result(@{@"textureId" : @(textureId)});
 }
 
+- (void)addObservers:(BetterPlayer*) player {
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(itemDidPlayToEndTime:)
+                                                 name:AVPlayerItemDidPlayToEndTimeNotification
+                                               object:[player player].currentItem];
+}
+
+- (void)itemDidPlayToEndTime:(NSNotification*) notification {
+    // Delay 1 second for users to see the video is finished.
+    // - Without this delay, the progress bar will run like "-0:01" then everything disappear.
+    // - With this delay, the progress bar will run like "-0:01" then "-0:00"
+    // and Pause button switched to Play button(it mean the player had stopped)
+    // then clear all info.
+    dispatch_after(dispatch_time(DISPATCH_TIME_NOW, (int64_t)(1 * NSEC_PER_SEC)),
+                   dispatch_get_main_queue(), ^{
+        // clear nowPlayingInfo
+        [MPNowPlayingInfoCenter defaultCenter].nowPlayingInfo = @{};
```

Comment:
So far no need to remove the content info itself, [just disable playback control is fine](https://docs.google.com/spreadsheets/d/1iXu4Vz9wYpT6fHE7oK_W4vyHP_-BGTvk7HTcETFJ_0I/edit#gid=2109148807&range=120:120)

Is it okay to delete this line to keep content info?
If it affect some logic, llet me know.

Reply:
@HiroshiOshiro 
If we delete this line, we should disable seek bar too,
but it will affect other disable seek bar logic, and we need to test it again,
if you fine with those things, i will implement it then.

Reply:
@mvn-hienvu-hn 

> but it will affect other disable seek bar logic, and we need to test it again,

How long do you think it will take?

Reply:
@HiroshiOshiro 
I think it could take about 4hrs for both implement and test (in the case that no issue occur).

Reply:
@HiroshiOshiro cc @mvn-thanhluu-hn 
I implemented your suggestion, could you please review it again 🙇?